src/features/pendo/Pendo.jsx

```diff
@@ -26,14 +26,19 @@ function pendoHelperUser(pendoKey) {
 }
 
 // Use this function for a custom visitor ID
-function pendoHelperCustom(pendoKey) {
+async function pendoHelperCustom(pendoKey) {
   // eslint-disable-next-line prefer-const
-  let visitor = null;
+  const { username } = getAuthenticatedUser();
```

Comment:
During my tests when upgrading to Quince, I saw that getAuthenticatedUser() can return null, in that case, this breaks.